README.md

```diff
@@ -286,6 +286,11 @@ Here are all the available options:
     <td>If true default values based on the "type" of the property will be used</td>
     <td><code>true</code></td>
   </tr>
+  <tr>
+    <td>validation_error_placement</td>
+    <td>Change position where errors are shown. Values are oneOf['grouped', 'below_field']</td>
+    <td><code>grouped</code></td>
```

Comment:
The current default is `below_field` I think.

Reply:
currently errors are grouped outside the fields. 

![Schermata 2020-11-02 alle 12 28 34](https://user-images.githubusercontent.com/22480239/97865765-fefffd80-1d0a-11eb-8aa2-0e3b73432885.png)

with the "validation_error_placement" with value "below_field" I get this

![Schermata 2020-11-02 alle 12 59 48](https://user-images.githubusercontent.com/22480239/97865964-556d3c00-1d0b-11eb-8b1d-93f7cb11cf99.png)

Reply:
Would be nice, if they are grouped in one "error-box", in ie. a `<LI>`st

Reply:
Currently the errors are grouped like this (for each area)

![Schermata 2020-11-02 alle 16 32 41](https://user-images.githubusercontent.com/22480239/97886588-11892f80-1d29-11eb-8c34-16035adb2641.png)